src/xngin/apiserver/routers/common_api_types.py

```diff
@@ -985,6 +985,10 @@ class Assignment(ApiBaseModel):
             max_length=MAX_NUMBER_OF_CONTEXTS,
         ),
     ] = None
+    current_mu: list[float] | None = None
+    current_covariance: list[list[float]] | None = None
+    current_alpha: float | None = None
+    current_beta: float | None = None
```

Comment:
Do these params really need to be returned back with assignments?
I'd expect most consumers wouldn't need these details (unless you want to debug an individual draw)?